setup.cfg

```diff
@@ -1,4 +1,4 @@
 [tool:pytest]
-addopts = -vs --tb=long --showlocals --html=ui-test.html --self-contained-html
```

Comment:
Apparently the -s cli option will prevent log captures in the latest pytest-html releases